docs/install.md

```diff
@@ -150,7 +150,16 @@ Select the following tabs for the installation description of your target.
     <font size="+2">3. First boot of the new VirtualBox image</font>
 
     Press the start button (green arrow) to 'boot up' your system based on the DietPi image.
-    If you do not have a wired LAN connection you have to change the network settings matching to your environment (files `\boot\dietpi.txt` and `\boot\dietpi-wifi.txt`).
+
+    <font size="+1">3.1 Configuration hints: Disable IPv6 when the host uses WiFi</font>
```

Comment:
Probably we can move this into a collapsed warning?
```
=== warning "You must disable IPv6 when the host uses WiFi"
```

A reboot is btw not required, disabling IPv6 takes effect immediately. So from the first related error prompt (the first attempted IPv6 connection), one can directly open `dietpi-config` (no need to open a subshell first, as there is a dietpi-config menu entry) > Network Options: Adapters > IPv6.

Reply:
Ok changed.
Opening a shell is needed because you typically run into problems during the first boot time. You open the subshell out of the init procedure.